### PR TITLE
Add Cx-Flow Tests

### PR DESCRIPTION
**Pull request by dbolharovcmx**

### CODE REVIEW

**Comment by dbolharovcmx** [comment]
[Add tests for CxFlow in SCA Agent](https://trello.com/c/HGJGXfve/1111-add-tests-for-cxflow-in-sca-agent)

File: tests/README.md [hunk 1]
```diff
@@ -97,6 +97,8 @@ ___
 
 ### Running Testing Scenarios Locally
 
+**!!! IMPORTANT !!! Do not edit run_test_scenario.sh in any Windows Editor**
+
```
Comment (maor-rosenfeld) [inline tests/README.md:101]: read about .gitattributes

Comment (dbolharovcmx) [reply tests/README.md:101]: **.gitattributes** configured as expecpected right now and is just fine. But when folks are working locally and do the modifications in that script in Windows - it appears that everything works not as expected. That line is just an attempt to warn the reader about that and save him some time.

File: tests/src/connectivity/scripts/cxflow_tests.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import time
+from test_base import ConnectivityBase
+
+
+class CxFlowTests(ConnectivityBase):
+    __test__ = True
+
+    def test_cxflow_connectivity(self):
+        # Wait for CxFlow to wake up correctly - about that time for secure startup
+        time.sleep(60)
```
Comment (Snirx1) [inline tests/src/connectivity/scripts/cxflow_tests.py:11]: maybe worth calling it's health check if it has one

File: tests/README.md (hunk 1 above)
Comment (Snirx1) [reply tests/README.md:101]: need something that explain the problem better, also where does the script breaks in that case? what command?

**Review by Snirx1 — CHANGES_REQUESTED** [review]

**Review by Snirx1 — APPROVED** [review]

**Pull request merged**